src/components/breadcrumbs/crumb/crumb.component.tsx

```diff
@@ -24,43 +24,54 @@ export interface CrumbProps
       | "disabled"
     >,
     TagProps {
+  /** This sets the Crumb to current, does not render Link */
   isCurrent?: boolean;
-  onClick?: (
-    ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>,
-  ) => void;
 }
 
-const Crumb = React.forwardRef<HTMLLIElement, CrumbProps>(
-  ({ href, isCurrent, children, onClick, ...rest }, ref) => {
+function mergeRefs<T>(...refs: Array<React.Ref<T> | undefined>) {
+  return (node: T) => {
+    refs.forEach((ref) => {
+      if (!ref) return;
+      if (typeof ref === "function") {
+        ref(node);
+      } else {
+        (ref as React.MutableRefObject<T | null>).current = node;
+      }
+    });
+  };
+}
+
+const Crumb = React.forwardRef<HTMLElement, CrumbProps>(
+  ({ href, isCurrent, children, onClick, ...rest }, forwardedRef) => {
     const { isDarkBackground } = useBreadcrumbsContext();
+    const crumbRef = React.useRef<HTMLElement | null>(null);
 
-    const isSafari = React.useMemo(() => {
-      if (typeof navigator === "undefined") return false;
-      return /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+    const focusCrumb = React.useCallback(() => {
+      try {
+        crumbRef.current?.focus?.({ preventScroll: true } as FocusOptions);
+      } catch {
+        crumbRef.current?.focus?.();
+      }
     }, []);
 
+    type LinkOnClick = NonNullable<LinkProps["onClick"]>;
```

Comment:
comment: we don't need these types, they also shouldn't be initialised in a function body